dependencies.yaml

```diff
@@ -122,3 +138,14 @@ dependencies:
           - sphinxcontrib-websupport
           - sphinx-markdown-tables
           - sphinx-copybutton
+  test_python:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - networkx>=2.5.1
```

Comment:
We should add `networkx` back to the "build" dependencies above (as well as leave it here).

`networkx` should be considered a runtime dependency of the package, see https://github.com/rapidsai/cugraph/blob/df12979cd25f22b28c52417fc3b3a6a55fcdfc57/python/cugraph/cugraph/experimental/compat/nx/Graph.py#L14

This was also discussed in [Slack](https://nvidia.slack.com/archives/CALREMR0T/p1671132367806729?thread_ts=1671128173.982389&cid=CALREMR0T) and resulted in https://github.com/rapidsai/cugraph/issues/3084.

cc: @rlratzel

Reply:
fixed in 9608f16